### DISCUSSION TITLE
subscribe without google on android

### DISCUSSION THREAD

**Discussion by m4ri0g** [Docile-Alligator/Infinity-For-Reddit#1718] [Help needed] [answered]
You offer the possibility to download the app from f-droid repositories or even install the apk manually. 
however it is not possible to subscribe on a degoogled device 

are there way to subscribe without passing through google play/ google pay. 

I installed infiny on a graphene os without google services but i cannot subscribe.. and even if i would use another device to subscribe i cannot open the link restore purchases. 
i am willing to support the project but installing google play and provide payment information to google is a no go for me. 

i am sure i am not the only one in this situation, am I?

**Comment by Docile-Alligator** [comment, marked as answer]
Hi thank you for your support. You can download Patreon version here: https://github.com/Docile-Alligator/Infinity-For-Reddit/releases/tag/v7.3.3

It does not require Google services.
